src/Tizen.NUI/src/internal/WebView/WebContext.cs

```diff
@@ -115,6 +115,39 @@ protected override void Dispose(DisposeTypes type)
         [UnmanagedFunctionPointer(CallingConvention.StdCall)]
         private delegate void WebContextPasswordDataListAcquiredProxyCallback(IntPtr list);
 
+        [UnmanagedFunctionPointer(CallingConvention.StdCall)]
+        private delegate void WebContextHttpRequestInterceptedProxyCallback(IntPtr interceptor);
+
+        private EventHandler<WebContextHttpRequestInterceptedEventArgs> httpRequestInterceptedEventHandler;
+        private WebContextHttpRequestInterceptedProxyCallback httpRequestInterceptedCallback;
+
+        /// <summary>
+        /// Event for the HttpRequestIntercepted signal which can be used to subscribe or unsubscribe the event handler.<br />
+        /// This signal is emitted when http request would be intercepted.<br />
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public event EventHandler<WebContextHttpRequestInterceptedEventArgs> HttpRequestIntercepted
```

Comment:
I think, Delegate setter is more proper, because http request interceptor is only allow one handler, if multiple event handler was react with this event it could be wrong

How about think it?

```suggestion
        public SetHttpRequestInterceptorDelegate(Action<WebHttpRequestInterceptor> action) {
                    httpRequestInterceptedCallback = (IntPtr interceptor) => {
                        action(WebHttpRequestInterceptor(interceptor, true));
                    }
                    IntPtr ip = Marshal.GetFunctionPointerForDelegate(httpRequestInterceptedCallback);
                    Interop.WebContext.RegisterRequestInterceptedCallback(SwigCPtr, new HandleRef(this, ip));
        }
```

Reply:
And I think you need to keep a consistency with Tizen.WebView
https://github.com/Samsung/TizenFX/blob/8bee3b086da6909d53361da7b0e2b1610d0fbdd9/src/Tizen.WebView/Tizen.WebView/Context.cs#L157

Reply:
I replaced event handler with delegate, but I followed naming rule of other APIs of NUI WebView, and renamed it to RegisterHttpRequestedCallback.
What do you think?

Reply:
Ok, It also good